tree/dataframe/inc/ROOT/RDF/GraphNode.hxx

```diff
@@ -151,8 +151,8 @@ public:
    void SetExplored() { fIsExplored = true; }
 
    ////////////////////////////////////////////////////////////////////////////
-   /// \brief The node is considered just created
-   void SetIsNew(bool isNew) { fIsNew = isNew; }
+   /// \brief Mark this node as "not newly created"
```

Comment:
clang-tools complains but never says why, I speculate it comes from using `"` inside of a comment

Reply:
lol what's wrong with double quotes in comments? :smile: 

as you see from the log, the script messed up the git revision, it's a bug in the check script:

```
fatal: bad object 466d814583c15aa3ff8cc095e410522420f8c560
```

Reply:
Well, afair some combinations of comment symbols `///` followed by quotes seem to mess such checks. At least I do not see anything else that could have broken the check script

Reply:
It's picking up a bad git commit, it's a bug somewhere in the check script. the formatting is fine:

```
~/R/master (df-graphutils<>) git checkout df-graphutils && git rebase master && git clang-format --diff master df-graphutils
Already on 'df-graphutils'
Your branch and 'origin/df-graphutils' have diverged,
and have 52 and 15 different commits each, respectively.
  (use "git pull" to merge the remote branch into yours)
Current branch df-graphutils is up to date.
```